### ISSUE TITLE
Crawled Property for Document Library

### ISSUE THREAD

**Issue by aavi92** [microsoft-search/pnp-modern-search#2614]
Is there a Crawled Property for Document Library Name which can be used as a Filter ???

**Comment by kasperbolarsen** [comment]
On the road right now, so it is the short version: I would expect the name of the library is the same as Title. In order to use it as a refiner you will need to map Title to a refinablestring as Title is not sortable out of the box.

**Comment by FranckyC** [comment]
You're right @kasperbolarsen, you can use the 'Title' property mapped with a RefinbleString.
